Misc/NEWS.d/next/Core and Builtins/2020-01-25-23-51-17.bpo-39453.xCOkYk.rst

```diff
@@ -0,0 +1,2 @@
+Fix contains method of list to hold strong references of elements while
```

Comment:
I'm curious how much this affects or doesn't affect the search time for a long list.

Reply:
@rhettinger 
AS-IS
```bash
./python.exe -m pyperf timeit "l = [1] * 99999" "3 in l"
.....................
Mean +- std dev: 4.53 ms +- 0.22 m
```

TO-BE
```bash
./python.exe -m pyperf timeit "l = [1] * 99999" "3 in l"
.....................
Mean +- std dev: 5.17 ms +- 0.21 ms
```